File: Java/3224.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner scanner = new Scanner(System.in);
		
		String j = scanner.next();
		String d = scanner.next();
		
        System.out.println(j.length() >= d.length() ? "go" : "no");
	}
}

File: Kotlin/3224.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`);
    val jon = scanner.next();
    val doc = scanner.next();
    if (jon.length >= doc.length) {
        println("go")
    } else {
        println("no")
    }
}
